s4e-backend/pom.xml

```diff
@@ -26,6 +26,7 @@
         <greenmail.version>1.5.11</greenmail.version>
         <slugify.version>2.4</slugify.version>
         <springdoc-openapi.version>1.2.17</springdoc-openapi.version>
+        <awssdk.version>2.10.32</awssdk.version>
```

Comment:
why this version?

Reply:
It was the latest when I added the dependency. They have a short release cycle